active/radiobuttons/RadioButtonsAPI.md

```diff
@@ -0,0 +1,182 @@
+
+    Windows.UI.Xaml.DependencyObject ContainerFromItem(Object item);
+    Windows.UI.Xaml.DependencyObject ContainerFromIndex(Int32 index);
+
+    Int32 SelectedIndex;
+    Object SelectedItem;
+    event Windows.UI.Xaml.Controls.SelectionChangedEventHandler SelectionChanged;
+
+    Int32 MaxColumns;
+    Object Header;
+
+    static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedIndexProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedItemProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxColumnsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
+}
+}
+```
+
+## RadioButtons Automation Peer
+
+```
+namespace Microsoft.UI.Xaml.Automation.Peers
+{
+[webhosthidden]
+unsealed runtimeclass RadioButtonsListViewItemAutomationPeer : Windows.UI.Xaml.Automation.Peers.ListViewItemAutomationPeer
+{
+    RadioButtonsListViewItemAutomationPeer(RadioButtonsListViewItem owner);
+}
+}
+```
+
+## RadioButtons Primitives
+
+```
+namespace Microsoft.UI.Xaml.Controls.Primitives
+{
+[webhosthidden]
+unsealed runtimeclass RadioButtonsListView : Windows.UI.Xaml.Controls.ListView
+{
+    RadioButtonsListView();
+}
+
+[webhosthidden]
+[default_interface]
+unsealed runtimeclass RadioButtonsListViewItem : Windows.UI.Xaml.Controls.ListViewItem
+{
+    RadioButtonsListViewItem();
+}
+}
+```
+
+# API Notes
+
+| Name | Description |
+|:-:|:--|
+| Header | Places a text label above the group of RadioButton elements and is exposed to the UIA tree via the DataContext property on the RadioButtons control's ContentPresenter. |
+| HeaderTemplate | The template that specifies the visualization of the header object |
+| SelectedItem | Gets or sets the index of the selected item. Selection is denoted by that item's RadioButton icon being checked. |
+| SelectedIndex | Gets or sets the selected item.|
+| Items | Gets or sets an object source used to generate the content of the ItemsControl. All items placed within the RadioButtons control will get the content of a generated RadioButton. |
+| ItemsSource | Gets or sets an object source used to generate the content of the ItemsControl. |
+| ItemTemplate | The template set on any generated RadioButton elements within the RadioButtons control. If the root of the ItemTemplate is a RadioButton it will be used as the generated container. |
+| MaxColumns | Defines the number of columns to divide the RadioButton group items into. |
+
+## RadioButton Elements in the RadioButtons controls
+Properties set on a RadioButton that is placed in a RadioButtons control are ignored by the narrator.
+
+For example, in this scenario:
+
+```xaml
+<muxc:RadioButtons Header="App Mode">
+    <RadioButton GroupName="group test">test 1</RadioButton>
```

Comment:
It definitely wasn't clear to me from earlier in the spec that this scenario (adding RadioButton controls into RadioButtons control) was supported. Not sure how that can be clarified earlier.

Reply:
This should be in the examples above, *however* there appears to be a bug with it and I need @teaP to help address them before I add more examples to this spec.

Reply:
This scenario is not supported. Is this a new requirement?

Reply:
It should be now with the migration from ListView to ItemsRepeater. Resolving this.